packages/components/src/components/hds/side-nav/index.ts

```diff
@@ -51,9 +51,8 @@ export interface HdsSideNavSignature {
 }
 
 export default class HdsSideNav extends Component<HdsSideNavSignature> {
-  @tracked isResponsive = this.args.isResponsive ?? true; // controls if the component reacts to viewport changes
-  @tracked isMinimized = this.args.isMinimized ?? false; // sets the default state on 'desktop' viewports
-  @tracked isCollapsible = this.args.isCollapsible ?? false; // controls if users can collapse the sidenav on 'desktop' viewports
+  @tracked _isControlled = this.args.isMinimized !== undefined;
```

Comment:
[question] Do we expect this to be controlled from the outside? (that seems strange to me)

Reply:
Also, how do we make sure we don't follow in the same problem as the one described above, for the masked input?

Reply:
Resolving this because we ended up removing the controlled logic from this PR.